src/YetAnotherHttpHandler.Unity/Assets/Editor/Scripts/PackageExporter.cs

```diff
@@ -1,34 +1,67 @@
 #if UNITY_EDITOR
 
+using Needle.HybridPackages;
 using System;
+using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using UnityEditor;
-using UnityEngine;
+using UnityEditor.PackageManager.Requests;
 
 public static class PackageExporter
 {
+    private static ListRequest s_listRequest;
+
     [MenuItem("Tools/Export Unitypackage")]
     public static void Export()
     {
-        PackDependencies("Cysharp.Net.Http.YetAnotherHttpHandler.Dependencies", "YetAnotherHttpHandler.deps.txt");
-        PackDependencies("Grpc.Net.Client.Dependencies", "Grpc.Net.Client.deps.txt");
+        EditorUtility.DisplayProgressBar("Packges", "Exporting packages, please wait...", 0);
+
+        s_listRequest = UnityEditor.PackageManager.Client.List(offlineMode: false, includeIndirectDependencies: true);
+
+        EditorApplication.update += OnEditorUpdate;
+    }
+
+    private static void OnEditorUpdate()
+    {
+        if (s_listRequest == null || !s_listRequest.IsCompleted)
+        {
+            return;
+        }
+
+        EditorApplication.update -= OnEditorUpdate;
+
+        ExportPackage("com.cysharp.yetanotherhttphandler", "Cysharp.Net.Http.YetAnotherHttpHandler.Dependencies");
+        ExportPackage("org.nuget.grpc.net.client", "Grpc.Net.Client.Dependencies");
```

Comment:
It seems like `Grpc.Net.Client` itself is not included. I apologize if my package naming or description was poor, but I intended that `Grpc.Net.Client.Dependencies.unitypackage` should include `Grpc.Net.Client` as well.

Reply:
Done! I didn't realize that in the case of Grpc's package he had to include himself.